### PR TITLE
Run on Xcode 10.2 on CI

### PR DESCRIPTION
**Pull request by jpsim**
This was just added to Azure Pipelines: https://github.com/Microsoft/azure-pipelines-image-generation/issues/815#issuecomment-482304766

### CODE REVIEW

**Comment by jpsim** [comment]
Need to fix the hanging performance test on Xcode 10.2.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -74,12 +63,85 @@ jobs:
...
+        YAMS_DEFAULT_ENCODING: UTF8
+    - task: PublishTestResults@2
+      inputs:
+        testRunner: JUnit
+        testResultsFiles: build/reports/**
+      condition: succeededOrFailed()
+
+- job: Xcode10
+  pool:
+    vmImage: 'macOS 10.14'
```
Comment (norio-nomura) [inline azure-pipelines.yml]: `vmImage` can be made a parameter like https://github.com/norio-nomura/Base32/pull/47/commits/20fcc9b2957f63e54d3ed3cdc05fbb01513ce4a8

Comment (jpsim) [reply azure-pipelines.yml]: 😻

**Comment by norio-nomura** [comment]
> Need to fix the hanging performance test on Xcode 10.2.

This can not be reproduced locally. 🤔 
Can you try rebuild?

**Comment by jpsim** [comment]
> This can not be reproduced locally. 🤔
> Can you try rebuild?

Happened again on rebuild. I think the problem is that there may not be any 32 bit simulators installed, or maybe they're not installed properly, and we're only seeing it hang for performance tests because the other ones don't require booting the simulator because they're just logic tests?

**Pull request merged**